web_pivot_computed_measure/static/src/scss/web_pivot_computed_measure.scss

```diff
@@ -0,0 +1,8 @@
+#add_computed_measure_wrapper {
+    padding: 0 20px;
+    min-width: 300px;
```

Comment:
You can use the 'o-search-options-dropdown-custom-item' mixin.

Use 'white-space: nowrap' to force text in a single line :)

Reply:
Thanks @Tardo , I added 'white-space: nowrap but not 'o-search-options-dropdown-custom-item' mixin because it restricts the element to have a maximum width of 250px and that is not what I want.